torchci/pages/metrics.tsx

```diff
@@ -209,11 +214,97 @@ function TimePicker({ label, value, setValue }: any) {
         onChange={(newValue) => {
           setValue(newValue);
         }}
+        disabled={disabled}
```

Comment:
does this disabled value ever get used?

Reply:
nope, I'll clean it up. I originally had the time pickers in a disabled state when the user had a pre-defined time range selected, but it seemed less confusing to just hide them entirely.